src/api/configure.ts

```diff
@@ -0,0 +1,102 @@
+import { Request, Response } from 'express';
+
+import {
+    CallResponseHandler,
+    newErrorCallResponseWithMessage,
+    newFormCallResponse,
+    newOKCallResponse,
+    newOKCallResponseWithData,
+    newOKCallResponseWithMarkdown,
+} from '../utils/call-responses';
+import { AppCallRequest, AppCallResponse, Oauth2App } from '../types';
+import { hyperlink } from '../utils/markdown';
+import { pagerDutyConfigForm, pagerDutyConfigSubmit } from '../forms/configure-admin-account';
+import { oauth2Complete, oauth2Connect, oauth2Disconnect } from '../forms/oauth';
+import { isConnected, showMessageToMattermost } from '../utils/utils';
+import { configureI18n } from '../utils/translations';
+
+export const configureAdminAccountForm: CallResponseHandler = async (req: Request, res: Response) => {
```

Comment:
Optional

Should this `CallResponseHandler` be used on every handler? Also since the function is typed, you don't need to specify the arg types. I just tested this out, and typescript actually doesn't infer the specific request and response based if you specify it this way.

I suggest doing this otherwise the `CallResponseHandler` doesn't help at all:

```suggestion
export const configureAdminAccountForm: CallResponseHandler = async (req, res) => {
```